pkg/services/entities/api.go

```diff
@@ -0,0 +1,65 @@
+package entities
+
+import (
+	"errors"
+
+	"github.com/go-macaron/binding"
+
+	"github.com/grafana/grafana/pkg/api/response"
+	"github.com/grafana/grafana/pkg/api/routing"
+	"github.com/grafana/grafana/pkg/middleware"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/util"
+)
+
+func (e *EntityService) registerAPIEndpoints() {
+	if !e.IsEnabled() {
+		return
+	}
+
+	e.RouteRegister.Group("/api/entities", func(entities routing.RouteRegister) {
+		entities.Post("/:kind/", middleware.ReqSignedIn, binding.Bind(createEntityCommand{}), routing.Wrap(e.createHandler))
```

Comment:
Should we have a root path instead and include `kind` in the command instead?

Reply:
What do you think @marefr  @aknuds1? Root path with kind as parameter in command or `/:kind/ `?

Reply:
We'll have 2 kinds to start with Panels (former Library Panels) and Variables

Reply:
I decided to go with `root` path and passing kind as a param after discussions with Torkel.